condor.go

```diff
@@ -279,6 +287,14 @@ func (cl *CondorLauncher) handleLaunchRequests(condorPath, condorConfig string)
 				if err != nil {
 					log.Errorf("%+v\n", errors.Wrap(err, "failed to publish successful launch job update"))
 				}
+
+				if err := delivery.Ack(false); err != nil {
+					log.Error(errors.Wrap(err, "failed to ACK amqp Launch request delivery"))
+				}
+			}
+		default:
```

Comment:
It's presumably not going to be common, but if we're adding this default case I think we should add a log that we got an unrecognized message too.

Reply:
That's a good idea 👍